docs/_articles/en/user-guide/deploy-on-save.md

```diff
@@ -31,7 +31,7 @@ When you enable deploy on save for your project:
 
 - If there isn’t an active deployment and no files are queued for deployment, a file save triggers an immediate deployment.
 
-If you enable deploy on save while working against a scratch org, the context of the extension is package development model. This means that every time you save a file, **SFDX: Push Source to Default Scratch Org** is initiated and runs `force:source:push` under the hood. In org development model that works with non-source-tracked-orgs, every file save initiates **SFDX: Deploy Source to Org**, which runs `force:source:deploy`.
+If you enable deploy on save while working against a source-tracked org, every time you save a file, **SFDX: Push Source to Default Org** is initiated and runs `force:source:push` under the hood. For non source-tracked orgs, every file save initiates **SFDX: Deploy Source to Org**, which runs `force:source:deploy`.
```

Comment:
@sbudhirajadoc - are the palette menu options still using the sfdx commands?

Reply:
yes they are...